cytominer_eval/tests/test_operations/test_hitk.py

```diff
@@ -0,0 +1,88 @@
+import os
+import random
+import pathlib
+import tempfile
+import numpy as np
+import pandas as pd
+
+
+from cytominer_eval.transform import metric_melt
+from cytominer_eval.operations import hitk
+
+
+random.seed(42)
+tmpdir = tempfile.gettempdir()
+
+# Load LINCS dataset
+example_file = "SQ00015054_normalized_feature_select.csv.gz"
+example_file = pathlib.Path(
+    "{file}/../../example_data/compound/{eg}".format(
+        file=os.path.dirname(__file__), eg=example_file
+    )
+)
+
+df = pd.read_csv(example_file)
+df["Metadata_moa"] = df["Metadata_moa"].fillna("unknown")
```

Comment:
Once you make the `hitk` function more modular, you will also need to test with a different metadata variable.

A bonus thing to do is to test the function if the metadata variable as provided doesn't exist _and_ if the metadata variable is the same for all rows.

Reply:
I will make an issue for this. I'd rather get the easy functionality through.